regolith/schemas.py

```diff
@@ -3822,7 +3907,22 @@
                                   f"Allowed values are {', '.join(PROJECTUM_STATUS)}",
                    "required": False,
                    "type": "string",
-                   "eallowed": PROJECTUM_STATUS}
+                   "eallowed": PROJECTUM_STATUS},
+        "ana_repo_url": {"description": "link to remote repository",
```

Comment:
let's make this a list and simply call it `other_urls` and in the description say, "for example, analysis or data repositories."